resources/i18n/nl_NL/fdmprinter.def.json.po

```diff
@@ -2514,19 +2514,19 @@ msgstr "Acceleratie Primepijler"
 
 msgctxt "prime_tower_brim_enable label"
 msgid "Prime Tower Base"
-msgstr ""
+msgstr "Basis van de Prime Tower"
```

Comment:
This isn't consistent with the previous translations, "Prime Tower" was always translated as "Primepijler". Although I do not like this translation I think we should keep consistent

```suggestion
msgstr "Basis van de Primepijler"
```